tests/tools/s3/resource_manifest.json

```diff
@@ -255,5 +255,122 @@
       "resource_name": "aarch64-initramfs.cpio",
       "relative_path": "kernel/"
     }
+  ],
+  "v5": [
+    {
+      "resource_type": "kernel",
+      "resource_name": "bzimage-hello-busybox-halt",
+      "relative_path": "kernel/",
+      "image_format": "bzimage",
+      "halt_after_boot": true,
+      "with_disk": false,
+      "platform_type": "x86_64"
+    },
+    {
+      "resource_type": "kernel",
+      "resource_name": "vmlinux-hello-busybox-halt",
+      "relative_path": "kernel/",
+      "image_format": "elf",
+      "halt_after_boot": true,
+      "with_disk": false,
+      "platform_type": "x86_64"
+    },
+    {
+      "resource_type": "kernel",
+      "resource_name": "pe-hello-busybox-halt",
+      "relative_path": "kernel/",
+      "image_format": "pe",
+      "halt_after_boot": true,
+      "with_disk": false,
+      "platform_type": "aarch64"
+    },
+    {
+      "resource_type": "kernel",
+      "resource_name": "vmlinux-hello-busybox",
+      "relative_path": "kernel/",
+      "image_format": "elf",
+      "halt_after_boot": false,
+      "with_disk": false,
+      "platform_type": "x86_64"
+    },
+    {
+      "resource_type": "kernel",
+      "resource_name": "bzimage-hello-busybox",
+      "relative_path": "kernel/",
+      "image_format": "bzimage",
+      "halt_after_boot": false,
+      "with_disk": false,
+      "platform_type": "x86_64"
+    },
+    {
+      "resource_type": "kernel",
+      "resource_name": "pe-hello-busybox",
+      "relative_path": "kernel/",
+      "image_format": "pe",
+      "halt_after_boot": false,
+      "with_disk": false,
+      "platform_type": "aarch64"
+    },
+    {
+      "resource_type": "kernel",
+      "resource_name": "bzimage-focal",
+      "relative_path": "kernel/",
+      "image_format": "bzimage",
+      "halt_after_boot": false,
+      "with_disk": true,
+      "platform_type": "x86_64",
+      "distro": "Ubuntu"
+    },
+    {
+      "resource_type": "kernel",
+      "resource_name": "vmlinux-focal",
+      "relative_path": "kernel/",
+      "image_format": "elf",
+      "halt_after_boot": false,
+      "with_disk": true,
+      "platform_type": "x86_64",
+      "distro": "Ubuntu"
+    },
+    {
+      "resource_type": "kernel",
+      "resource_name": "pe-focal",
+      "relative_path": "kernel/",
+      "image_format": "pe",
+      "halt_after_boot": false,
+      "with_disk": true,
+      "platform_type": "aarch64",
+      "distro": "Ubuntu"
+    },
+    {
+      "resource_type": "kernel",
+      "resource_name": "pe-focal-halt",
```

Comment:
Do we need this one? I see that on x86, we don't keep the focal resources that are halting.

Reply:
inside the make_test_resources.sh file it does make focal resources that are halting.

Reply:
Yeah, I know that but I think we didn't also upload them to S3. I was asking if we need them somewhere in the tests.

Reply:
We might need them in the future. It makes the upload process easier if all the files that are generated are also uploaded. Otherwise we need to update the script as to not generate them anymore. The aarch64 does not currently run all tests that run on x86_64. To stay on the safe side I would say to keep the pe-focal-halt image as well.

Reply:
should I remove them in a different commit, stating the reason we don't actually need them?
or in a previous commit itself

Reply:
Separate commit sounds good! It can be placed wherever, I don't have a strong preference about it.